server.js

```diff
@@ -1,16 +1,25 @@
 const express = require('express');
 const api = require('./api');
-const app = express();
-
-const port = process.env.PORT || 1337;
-
-// set up swagger-ui
 const swaggerUi = require('swagger-ui-express');
 const swaggerDocument = require('./openapi.json');
-app.use('/api/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
-// routes
-app.get('/api/v1/ping', api.pong);
+const createServer = (port) => {
+    const app = express();
+    
+    app.get('/v1/ping', api.pong);
+    app.use('/api/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+    const server = app.listen(port, () => console.log(`Server listening on port ${port}`));
+    return server;
+
+
+};
+
+if(require.main === module) {
```

Comment:
Hva er det vi sjekker her?

Reply:
her er måten jeg fant ut hvordan man bestemmer om modulen er kjørt direkte fra node eller om den er pulled in by require. Kanskje det finnes en bedre måte? Eventuelt dra det ut i en initiation-fil. Må ha tilgang til server-setup som modul for å kjøre testene mot det. Gjør det iallefall rimelig ryddig

Reply:
Ja ok. Ser jo ikke så gærnt ut egentlig. Foreslår du bare slenger en kommentar over som sier noe ala 

`// false if tests are running. `

Eller var det omvendt 😅?

Med en enkel kommentar slipper man å bruke for mye hjernekraft :).